### ISSUE TITLE
When moving with arrows in specific Word document  current page and section number are announced in random places. 

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#9504]
I cannot share the document publicly here. If someone would like to work on this just mention me here and I will send it privately.

### Steps to reproduce:
1. Open the Word document in question,
2. Start moving with the down arrow to the first group of empty lines.
3. Move back with the up arrow.

### Actual behavior:
NVDA randomly announces page 1 Section 1.
### Expected behavior:
NVDA should announce page and section numbers only when they changes
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA version alpha-17077,7f274eca

#### Windows version:
Windows 7 X64
#### Name and version of other software in use when reproducing the issue:
Word 2010 32-bit
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Back to the 2017.4 and the issue is present with every version.

**Comment by Brian1Gaff** [comment]
This also has some tie in with the way you can get stuck in a page unless 
you can make the zoom level bigger so the pages only fit one to a screen. It 
seems the way nvda moves from page 1 to page 2 is very flaky at the moment.


bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.

**Comment by Adriani90** [comment]
@lukaszgo1 how is this now in NVDA last alpha? I don't experience this issue but I might not be the best one to reproduce it because i cannot see when the pages and sections visually change. But at least they are reported everytime in the same place, meaning after or before same character.

**Comment by lukaszgo1** [comment]
This still occurs with latest NVDA alpha and this specific file - no idea what is so special about it yet.

**Comment by lukaszgo1** [comment]
Yes, this is still reproducible with MS Word Version 2303 (Build 16227.20280) from Office 365.
